src/core/lib/security/credentials/tls/grpc_tls_credentials_options.cc

```diff
@@ -92,6 +92,26 @@ int grpc_tls_credentials_options_set_cert_request_type(
   return 1;
 }
 
+int grpc_tls_credentials_options_set_server_verification_option(
+    grpc_tls_credentials_options* options,
+    grpc_tls_server_verification_option server_verification_option) {
+  if (options == nullptr) {
+    gpr_log(GPR_ERROR,
+            "Invalid nullptr arguments to "
+            "grpc_tls_credentials_options_set_server_verification_option()");
+    return 0;
+  }
+  if (server_verification_option != GRPC_TLS_SERVER_VERIFICATION &&
```

Comment:
@yihuazhang TlsCredentialOptions is for both server as well as client. For server it does not make sense. Right?

Reply:
Sorry, Server will send GRPC_TLS_SERVER_VERIFICATION in that case.